pkgs/by-name/ca/castxml/package.nix

```diff
@@ -45,14 +48,17 @@ stdenv.mkDerivation (finalAttrs: {
   ];
 
   cmakeFlags = [
-    "-DCLANG_RESOURCE_DIR=${libclang.dev}/"
-    "-DSPHINX_HTML=${if withHTML then "ON" else "OFF"}"
-    "-DSPHINX_MAN=${if withManual then "ON" else "OFF"}"
+    (lib.cmakeOptionType "path" "CLANG_RESOURCE_DIR" "${lib.getDev libclang}")
+    (lib.cmakeBool "SPHINX_HTML" withHTML)
+    (lib.cmakeBool "SPHINX_MAN" withManual)
   ];
 
   # 97% tests passed, 97 tests failed out of 2881
   # mostly because it checks command line and nix append -isystem and all
   doCheck = false;
+
+  strictDeps = true;
```

Comment:
adding `strictDeps = true;` broke the darwin builds.

[log](https://logs.ofborg.org/?key=nixos/nixpkgs.309698&attempt_id=c7edc306-e4cf-45ef-a894-0d736bd54824)

```
[ 12%] Building CXX object src/CMakeFiles/castxml.dir/castxml.cxx.o
In file included from /tmp/nix-build-castxml-0.6.6.drv-0/source/src/castxml.cxx:18:
/tmp/nix-build-castxml-0.6.6.drv-0/source/src/Options.h:19:10: fatal error: 'string' file not found
#include <string>
         ^~~~~~~~
1 error generated.
make[2]: *** [src/CMakeFiles/castxml.dir/build.make:76: src/CMakeFiles/castxml.dir/castxml.cxx.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:251: src/CMakeFiles/castxml.dir/all] Error 2
make: *** [Makefile:136: all] Error 2
error: boost::bad_format_string: format-string is ill-formed
```